test/common/testlib.py

```diff
@@ -1120,6 +1120,10 @@ def check_journal_messages(self, machine=None):
             # but we have to ignore that general header line
             self.allowed_messages.append('^Stack trace of.*')
 
+        if self.image == "fedora-33":
+            # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1871351
+            self.allowed_messages.append('.*type=1400.*avc:  denied  { getattr } for .* comm="cockpit-session" name="/".*')
```

Comment:
Nice, it's fixed in https://bodhi.fedoraproject.org/updates/FEDORA-2020-8f3381648b , so the refresh in a week should get it.